firmware/src/Conduit.cpp

```diff
@@ -12,6 +12,18 @@ device and decide what funciton to all is abstracted away entirely by this libra
 
 #include "Conduit.h"
 
+#define PREFIXED_NAME_CHARS   35
+#define STATIC_STRING_CHARS   64
+#define RECONNECT_DELAY_MS    5000
+#define CONNECT_DELAY_MS      500
+#define ROOT_NODE_NAME        "ROOT"
+#define DEVICE_PATH_SUFFIX    "/device"
```

Comment:
Yup agree it is much clearer, but figured it'd be nice to have a section of `defines` that can be touched and another for those that should not be :)